ui/src/submissions/conferences/containers/ExistingConferencesAlertContainer.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useMemo } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { Button } from 'antd';
+import FieldInfoAlert from '../../common/components/FieldInfoAlert';
+import pluralizeUnlessSingle from '../../../common/utils';
+import ExistingConferencesDrawer from '../components/ExistingConferencesDrawer';
+import { EXISTING_CONFERENCES_NS } from '../../../reducers/search';
+import {
+  RANGE_AGGREGATION_SELECTION_SEPARATOR,
+  START_DATE_ALL,
+} from '../../../common/constants';
+import { searchQueryUpdate } from '../../../actions/search';
+
+export function ExistingConferencesAlert({
+  dispatch,
+  dates,
+  numberOfConferences,
+}) {
+  const [openingDate, closingDate] = dates;
+  const [showDrawer, setShowDrawer] = useState(false);
+
+  useMemo(
+    () => {
+      if (openingDate && closingDate) {
+        dispatch(
+          searchQueryUpdate(EXISTING_CONFERENCES_NS, {
+            contains: `${openingDate}${RANGE_AGGREGATION_SELECTION_SEPARATOR}${closingDate}`,
+            start_date: START_DATE_ALL,
+          })
+        );
+      }
+    },
+    [dispatch, openingDate, closingDate]
+  );
+
+  return (
+    numberOfConferences > 0 && (
+      <>
+        <FieldInfoAlert
+          description={
+            <span>
+              <strong data-test-id="conferences-exist-alert-number">
+                {numberOfConferences}
+              </strong>{' '}
+              other {pluralizeUnlessSingle('conference', numberOfConferences)}{' '}
+              found in these dates.
+              <Button
+                type="link"
+                size="small"
+                onClick={() => setShowDrawer(true)}
+              >
+                Show more
+              </Button>
+            </span>
+          }
+        />
+        <ExistingConferencesDrawer
+          visible={showDrawer}
+          onClose={() => setShowDrawer(false)}
+          numberOfConferences={numberOfConferences}
+        />
+      </>
+    )
+  );
+}
+
+ExistingConferencesAlert.propTypes = {
+  dispatch: PropTypes.func.isRequired,
+  dates: PropTypes.arrayOf(PropTypes.string).isRequired,
+  numberOfConferences: PropTypes.number,
+};
+
+const stateToProps = state => ({
+  numberOfConferences: state.search.getIn([
+    'namespaces',
+    EXISTING_CONFERENCES_NS,
+    'total',
+  ]),
+});
+
+const dispatchToProps = dispatch => ({ dispatch });
```

Comment:
instead of pass the `dispatch` you should pass a prop `onDatesChange` that dispatches the action then call this in the component (see other example containers).